database/models/InviteCode.js

```diff
@@ -17,7 +18,7 @@ module.exports = (sequelize) => {
       },
       code: {
         type: DataTypes.STRING,
-        defaultValue: () => new RandExp(/^[a-zA-Z0-9]{8}$/).gen(),
+        defaultValue: uid(),
```

Comment:
you need to parse a function to `defaultValue`, otherwise the invite code would be the same every invite generation which results in a DB validation error.
```suggestion
        defaultValue: () => uid(),
```